src/FINDOLOGIC/Export/XML/XMLItem.php

```diff
@@ -122,4 +124,22 @@ private function buildUsergroups(\DOMDocument $document)
 
         return $usergroups;
     }
+
+    private function validateImages(array $images){
+
+        $valid = false;
+
+        foreach ($images as $image) {
+            if($image->getType() === Image::TYPE_DEFAULT){
+                $valid = true;
+                break;
+            }
+        }
+
+        if (!$valid) {
+            throw new \FINDOLOGIC\Export\Data\BaseImageMissingException();
+        }
+
+        return $valid;
```

Comment:
Is a return value actually necessary if an exception is thrown? This will always be `true`.

Reply:
Done